src/redux/actions/group-actions.js

```diff
@@ -6,6 +6,14 @@ export const fetchGroups = (options = {}) => ({
   payload: GroupHelper.fetchGroups(options)
 });
 
+export const fetchSystemGroup = (filterValue) => ({
+  type: ActionTypes.FETCH_SYSTEM_GROUP,
+  payload: GroupHelper.fetchGroups({
+    name: filterValue !== '' ? filterValue : 'default',
```

Comment:
If the `filterValue` is undefined it will be used in here. Is that intentional? I think it would be better to use shortcutting.
```JS
name: filterValue || 'default'
```

Or even better
```JS
export const fetchSystemGroup = (filterValue = 'defaul') => ({
```